test/fw/doc/conf.py

```diff
@@ -246,7 +246,7 @@
 # (source start file, target name, title,
 #  author, documentclass [howto, manual, or own class]).
 latex_documents = [('index', 'PbsTestLab.tex', u'PbsTestLab Documentation',
-                    u'Copyright (C) 1994-2018 Altair Engineering, Inc',
```

Comment:
Travis is failing because the end quote was removed.

Reply:
There are two additional instances of this at lines 278 and 290 in this file.